python/GafferArnoldTest/IECoreArnoldPreviewTest/RendererTest.py

```diff
@@ -259,39 +271,37 @@ def testShaderNames( self ) :
 			self.assertEqual( len( shaderNames ), 4 )
 			self.assertEqual( len( [ i for i in shaderNames if i.split(":")[-1] == "myHandle" ] ), 2 )
 
-	def testShaderNodeConnectionType( self ) :
+	def testShaderConnections( self ) :
```

Comment:
@danieldresser-ie, this could do with your eyes. I've intentionally removed the support for connections to non-shader nodes that you introduced back in https://github.com/GafferHQ/gaffer/commit/584671517170422a2102788c00548fd79f1cf440#diff-7e15ae85c10bcfa25e451ada1ca4e068. I did this partly on the assumption that it's no longer needed (I may be wrong, but I think it was for early light filter support, and we've got a more official mechanism for that now), partly on the basis that it really doesn't make much sense, and partly on the basis that it allows further code cleanup. Can you think of any reason it's not OK?

On the plus side, I have been able to revive the commented test for connections to array shader parameters, because Arnold now includes ramp_rgb in the standard distribution.